### PR TITLE
Fixed bug for test of CASSANDRA-7816

### PR DESCRIPTION
**Pull request by stef1927**

### CODE REVIEW

**Comment by aboudreault** [comment]
Assigned myself as tester in 7816, will take a look at this asap.

**Comment by stef1927** [comment]
Great, thanks

File: pushed_notifications_test.py [hunk 1]
```diff
@@ -97,12 +101,11 @@ def restart_node_test(self):
         """        
 
         self.cluster.populate(2).start()
-
         node1, node2 = self.cluster.nodelist()
 
         waiter = NotificationWaiter(self, node1, "STATUS_CHANGE", num_notifications=2)
 
-        for i in range(3):
+        for i in range(5):
```
Comment (mambocab) [inline pushed_notifications_test.py]: Any particular reason to change this?

**Comment by mambocab** [comment]
I'm curious about https://github.com/riptano/cassandra-dtest/pull/179/files#r26330695, but +1

File: pushed_notifications_test.py (hunk 1 above)
Comment (stef1927) [inline pushed_notifications_test.py]: Yes occasionally I could not reproduce the multiple UP notifications with only 3 attempts and so I increased it to 5.

Comment (mambocab) [inline pushed_notifications_test.py]: I figured it was something like that. Thanks!

**Pull request merged**